src/sign.builds

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <PropertyGroup>
+    <OutDir>$(BinDir)</OutDir>
+  </PropertyGroup>
+
+  <UsingTask AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" TaskName="ReadSigningRequired" />
+
+  <!-- populates item group FilesToSign with the list of files to sign -->
+  <Target Name="GetFilesToSignItems">
+    <!-- find all of the "published" exe files -->
+    <ItemGroup>
+      <PublishedExeMarkerFile Include="$(OutDir)**\*.publishedexe" />
+    </ItemGroup>
+    <!-- the published exes will be renamed to dlls so rename the marker file to match -->
+    <Move SourceFiles="@(PublishedExeMarkerFile->'%(RootDir)%(Directory)%(Filename).exe.requires_signing')"
+          DestinationFiles="@(PublishedExeMarkerFile->'%(RootDir)%(Directory)%(Filename).dll.requires_signing')" />
+    <!-- now read all of the marker files and populate the FilesToSign item group -->
+    <ItemGroup>
+      <SignMarkerFile Include="$(OutDir)**\*.requires_signing" />
+    </ItemGroup>
+    <ReadSigningRequired MarkerFiles="@(SignMarkerFile)">
+      <Output TaskParameter="SigningMetadata" ItemName="FilesToSign" />
+    </ReadSigningRequired>
+  </Target>
+
+  <Target Name="Build"
+          Condition="'$(SkipSigning)' != 'true' and '$(SignType)' != 'oss'"
+          DependsOnTargets="GetFilesToSignItems">
+    <CallTarget Targets="SignFiles" />
```

Comment:
Oops I deleted it from sign.targets, I'll add it back to satisfy the case when the MicroBuild signing package isn't installed.